packages/mui-base/src/useTransition/useTransitionStateManager.ts

```diff
@@ -11,8 +12,14 @@ export type UseTransitionStateManagerReturnValue = {
    * Callback to be called when the element has completely exited.
    */
   onExited: () => void;
+  /**
+   * The current transition status.
+   */
+  transitionStatus: TransitionStatus;
 };
 
+export type TransitionStatus = 'initial' | 'opening' | 'closing' | 'closed';
```

Comment:
Does this file need to exist now? Seems to be outdated after we merged `useTransitionStatus`

Reply:
I reverted all the changes. This hook is used by the legacy components. We will remove it once the old components are refreshed.